docs/workflow_solutions/using_anaconda.md

```diff
@@ -260,6 +260,273 @@ Now we can be sure that the correct versions of the software will be installed o
+
+### Steps to Install TensorFlow CPU and GPU Versions Using Cheaha CLI
+
+1. Create a new environment that is compatible with supported tensorflow versions, use the below command to do this. For this tuorial we use Python 3.11.
+
+    ```bash
+
+    conda create -n tensorflow python=3.11
+
+    ```
+
+2. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version use the below command.
+
+    ```bash
+
+    pip install -—upgrade pip
+
+    ```
+
+3. Install TensorFlow with pip
+
+    ```bash
+
+    pip install tensorflow[and-cuda]
+
+    ```
+
+The image below shows an output that the TensorFlow library will utilize the available GPU.
+
+![TensorFlow GPU](images/tensor_gpu.png)
+
+### Update packages in an environment
+
+To ensure packages and their dependencies are all up to date, it is a best practice to regularly update installed packages, and libraries in your activated environment.
+
+```bash
+
+conda update -—all
+
+```
+
+### Deleting an Environment
+
+To delete an environment, use the following command. Remember to replace `<env>` with the existing environment name.
+
+```bash
+
+conda env remove —-name <env>
+
+```
+
+### Sharing your environment file
+
+To share your environment for collaboration, there are primarily 3 ways to export environments, the below commands show how to create environment files that can be shared for replication. Remember to replace `<env>` with the existing environment name.
+
+1. Cross-Platform Compatible
+
+    ```bash
+
+    conda env export --from-history > <env>.yml 
+
+    ```
+
+1. Platform + Package Specific
+
+Create .yml file to share, replace `<envname>` (represents the name of your environment) and `<env>` (represents the name of the file you want to export) with preferred names for file.
+
+```bash
+
+    conda env export <envname> > <env>.yml 
+
+```
+
+1. Platform + Package + Channel Specific
+
+    ```bash
+
+    conda list —-explicit > <env>.txt
+
+                OR
+
+    conda list —-explicit > <env>.yml
+
+    ```
+
+### Activating/Using a shared environment file
+
+To use an environment file shared with you, navigate into the folder where you have the .yml file saved. When in the folder, use use the command to create .yml file in your terminal, replace `<env>` with the environment file name.
+
+```bash
+
+conda env create -—file <env>.yml 
+
+```
+
+### Switching between environments
+
+Switching between two environments containing different libraries, packages, and dependencies can be carried out following the below steps.
+
+1. Activate an environment using the below command
+
+    ```bash
+
+    conda activate <env>
+
+    ```
+
+1. To switch the environment, you can do one of the below options using the below commands
+
+a. By using these commands, first deactivate the environment with. Then you activate the second environment with. replace `<env2>` with name of other environment.
```

Comment:
Markdown doesn't understand lettering. Indent these `a` and `b` and use numbering instead. You'll also need to indent the fenced code blocks as well.